canonical-websites/build.snapcraft.io
# Adds consistent Vanilla styles for footer

As #1068 is blocked due to header design, we can at least merge footer separately.

## Done

- added Vanilla styles for footer (footer patter, grid, etc)
- updated footer to new design
- fixed issues around sticky footer

## QA

- Check out this feature branch
- Run the site using the command ```npm start -- --env=environments/dev.env```
- View the site locally in your web browser at: [http://0.0.0.0:8000/](http://0.0.0.0:8000/)
- Make sure there is new footer
- Make sure 'Back to top' works
- Sign in
- Make sure footer sticks to the bottom of the screen even if there is little content


## Issue / Card

Related to similar update on snapcraft.io: https://github.com/canonical-websites/snapcraft.io/pull/377

## Screenshots
<img width="1228" alt="screen shot 2018-03-05 at 12 08 16" src="https://user-images.githubusercontent.com/83575/36972316-f96210c0-206e-11e8-8102-926c8faafc70.png">
